### PR TITLE
[CI] bump gvm 0.2.4 and fix windows tooling issues for Windows 7

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

- Fail fast if gvm is not installed correctly and add some traces
- Install `gvm` always.
- Configure temporary USERPROFILE to the workspace location.
- Use a `https://s3.us-east-1.amazonaws.com/deploy.andrewkroh.com/gvm/gvm-windows-amd64.exe` temporarily.
- [x] Update the version for gvm with the latest @andrewkroh will provide it :)

## Why is it important?

Detect issues earlier

## Known issues

- `gvm` does not install go correctly

```
level=info msg="ERROR: Cannot find C:\\Users\\jenkins\\.gvm\\versions\\go1.14.7.windows.amd64\\bin\\go.exe"
C:\Users\jenkins\workspace\Beats_beats_PR-22661\src\github.com\elastic\beats>DIR /A /S C:\Users\jenkins\.gvm\versions\go1.14.7.windows.amd64 
...
  Directory of C:\Users\jenkins\.gvm\versions\go1.14.7.windows.amd64\bin
 
 11/20/2020  12:21 PM    <DIR>          .
 11/20/2020  12:21 PM    <DIR>          ..
 11/20/2020  12:21 PM    <DIR>          tool
```

- installed `gvm`  got corrupted for windows-7-32 bits
- `gvm` with chocolate exits without installing mage.

```
[2020-11-23T16:52:15.639Z] C:\Users\jenkins\workspace\Beats_beats_PR-22661\src\github.com\elastic\beats>mage dumpVariables 
[2020-11-23T16:52:15.639Z] 'mage' is not recognized as an internal or external command,
[2020-11-23T16:52:15.639Z] operable program or batch file.
```

- after installing it with `choco` then go environment is not ready
```
[2020-11-23T19:29:20.115Z] go: downloading github.com/magefile/mage v1.10.0
[2020-11-23T19:29:20.372Z] go tool: no such tool "compile"
[2020-11-23T19:29:20.372Z] go tool: no such tool "compile"
```

## Screenshots

image

### CODE REVIEW

File: .ci/scripts/install-tools.bat [hunk 1]
```diff
@@ -16,22 +16,30 @@ mkdir %WORKSPACE%\bin
...
+    if EXIST %WORKSPACE%\bin\%GVM_BIN% (
+        %GVM_BIN% version
+    ) else (
+        REM gvm.exe has not been installed for some unknown reasons
+        exit /b 1
+    )
 )
-FOR /f "tokens=*" %%i IN ('"gvm.exe" use %GO_VERSION% --format=batch') DO %%i
+
+FOR /f "tokens=*" %%i IN ('"%GVM_BIN%" use %GO_VERSION% --format=batch') DO %%i
```
Comment (v1v) [inline .ci/scripts/install-tools.bat:42]: is `use ` the correct format for the gvm?

https://github.com/andrewkroh/gvm#installation vs https://github.com/moovweb/gvm#installing-go

Comment (v1v) [reply .ci/scripts/install-tools.bat:42]: yes, it's there https://github.com/andrewkroh/gvm/blob/master/cmd/gvm/use.go

**Comment by v1v** [comment]
jenkins run the tests please

**Review by cachedout — APPROVED** [review]

**Comment by v1v** [comment]
I managed to find the reason of those failures, not sure how it happens:


```
[2020-11-20T12:22:03.678Z]  Directory of C:\Users\jenkins\.gvm\versions\go1.14.7.windows.amd64\bin
[2020-11-20T12:22:03.678Z] 
[2020-11-20T12:22:03.678Z] 11/20/2020  12:21 PM    <DIR>          .
[2020-11-20T12:22:03.678Z] 11/20/2020  12:21 PM    <DIR>          ..
[2020-11-20T12:22:03.678Z] 11/20/2020  12:21 PM    <DIR>          tool
[2020-11-20T12:22:03.678Z]                0 File(s)              0 bytes
```

So the bin folder of the just installed go version is empty :S

**Comment by v1v** [comment]
jenkins run the tests please

**Comment by v1v** [comment]
jenkins run the tests please

**Comment by v1v** [comment]
jenkins run the tests please

**Comment by v1v** [comment]
jenkins run the tests please

**Comment by v1v** [comment]
jenkins run the tests please

**Comment by v1v** [comment]
jenkins run the tests please

**Review by kuisathaverat — APPROVED** [review]

**Pull request merged**